src/components/Toggle/index.tsx

```diff
@@ -0,0 +1,208 @@
+    width: 48,
+  },
+  small: {
+    ball: 12,
+    height: 20,
+    width: 40,
+  },
+} as const
+
+const StyledToggle = styled.div<{
+  'data-checked': boolean
+  'data-disabled': boolean
+  size: 'small' | 'large'
+}>`
+  box-sizing: content-box;
+  outline: none;
+  overflow: hidden;
+  display: flex;
+  align-items: center;
+  border: none;
+  border-radius: ${TOGGLE_RADIUS};
+  position: relative;
+  transition: all 300ms;
+  background-color: ${({ theme }) => theme.colors.neutral.textWeak};
+  width: ${({ size }) => SIZES[size].width}px;
+  height: ${({ size }) => SIZES[size].height}px;
+
+  &:after {
+    content: '';
+    position: absolute;
+    top: ${({ size }) => SIZES[size].height / 2 - SIZES[size].ball / 2}px;
+    left: 5px;
+    width: ${({ size }) => SIZES[size].ball}px;
+    height: ${({ size }) => SIZES[size].ball}px;
+    border-radius: ${TOGGLE_POINT_RADIUS};
+    background-color: ${({ theme }) => theme.colors.neutral.backgroundWeak};
+    transition: all 300ms;
+  }
+
+  &:focus-within,
+  &:focus {
+    box-shadow: ${({ theme }) => theme.shadows.focusPrimary};
+  }
+
+  &[data-disabled='false']:active:after {
+    width: ${({ size }) => SIZES[size].ball * 1.3775}px;
+  }
+
+  &[data-checked='true'] {
+    color: ${({ theme }) => theme.colors.neutral.textStrong};
+    background-color: ${({ theme }) => theme.colors.primary.backgroundStrong};
+
+    &:after {
+      left: calc(100% - 5px);
+      transform: translateX(-100%);
+    }
+
+    &:focus-within,
+    &:focus {
+      box-shadow: ${({ theme }) => theme.shadows.focusPrimary};
+    }
+  }
+
+  &[data-disabled='true'] {
+    background: ${({ theme }) => theme.colors.neutral.borderWeakDisabled};
+    &[data-checked='true'] {
+      background: ${({ theme }) =>
+        theme.colors.primary.backgroundStrongDisabled};
+    }
+  }
+`
+
+const StyledCheckbox = styled.input`
+  position: absolute;
+  opacity: 0;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  cursor: pointer;
+
+  &[disabled] {
+    cursor: not-allowed;
+  }
+`
+
+const StyledLabel = styled.label<{
+  size: 'small' | 'large'
+}>`
+  display: flex;
+  align-items: center;
+  cursor: pointer;
+
+  &:active ${StyledToggle}[data-disabled='false']:after {
+    width: ${({ size }) => SIZES[size].ball * 1.3775}px;
+  }
+
+  &[aria-disabled='true'] {
+    cursor: not-allowed;
+    color: ${({ theme }) => theme.colors.neutral.textDisabled};
+  }
+`
+
+const StyledLabelContent = styled.span<{
+  labelPosition: 'left' | 'right'
+}>`
+  ${({ theme, labelPosition }) =>
+    labelPosition === 'left'
+      ? `margin-right: ${theme.space['1']}`
+      : `margin-left: ${theme.space['1']}`}
+`
+
+const LabelContent = ({
+  label,
+  labelPosition,
+}: Required<Pick<ToggleProps, 'label' | 'labelPosition'>>) => {
+  if (typeof label === 'string') {
+    return (
+      <StyledLabelContent labelPosition={labelPosition}>
+        {label}
+      </StyledLabelContent>
+    )
+  }
+
+  return <>{label}</>
```

Comment:
Can't we return `label` directly?

```suggestion
  return label
```